src/main/java/com/gregtechceu/gtceu/core/MixinHelpers.java

```diff
@@ -213,8 +213,11 @@ public static void generateGTDynamicLoot(Map<ResourceLocation, LootTable> lootTa
                                     LootItem.lootTableItem(dropItem.getItem())
                                             .apply(SetItemCountFunction
                                                     .setCount(ConstantValue.exactly(oreMultiplier)))));
-                    // .apply(ApplyBonusCount.addOreBonusCount(Enchantments.BLOCK_FORTUNE)))); //disable fortune for
-                    // balance reasons. (for now, until we can think of a better solution.)
+                    if (ConfigHolder.INSTANCE.gameplay.oreFortuneDrops) {
+                        builder.apply(ApplyBonusCount.addOreBonusCount(Enchantments.BLOCK_FORTUNE)); // disable fortune
+                                                                                                     // by
+                        // default for balance reasons. (for now, until we can think of a better solution.)
```

Comment:
could you move this comment to the line above so it doesn't split like this?

Reply:
Wuh spotless jank spotted, fixed it